### PR TITLE
Fixing broken links in the java manual

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-drivers-385.surge.sh

File: java-manual/modules/ROOT/pages/cypher-workflow.adoc [hunk 1]
```diff
@@ -158,7 +158,7 @@ need.
 // include::{java-examples}/ReadingValuesExample.java[tags=java-driver-reading-values-null]
 // ----
 
-For more information, see https://neo4j.com/docs/api/java-driver/{java-driver-version}/org/neo4j/driver/Value.html[Java Driver API documentation -> Interface Value].
+For more information, see link:{neo4j-docs-base-uri}/api/java-driver/current/org.neo4j.driver/org/neo4j/driver/Value.html[Java Driver API documentation -> Interface Value].
```
Comment (stefano-ottolenghi) [inline java-manual/modules/ROOT/pages/cypher-workflow.adoc:161]: Now the link renders, but is still a 404. The current URL is https://neo4j.com/docs/api/java-driver/current/org.neo4j.driver/org/neo4j/driver/Value.html :)

Comment (lidiazuin) [reply java-manual/modules/ROOT/pages/cypher-workflow.adoc:161]: I don't understand? It's opening correctly here?

Comment (stefano-ottolenghi) [reply java-manual/modules/ROOT/pages/cypher-workflow.adoc:161]: IDK what I checked, sorry, my mistake :facepalm:

**Pull request merged**